src/www/ui_tests/api/Controllers/UploadControllerTest.php

```diff
@@ -1183,50 +1183,69 @@ private function testGetUploadLicensesPendingScan($version = ApiVersion::V2)
   }
 
   /**
-   * @test
-   * -# Test for UploadController::updateUpload()
-   * -# Check if response status is 202
-   */
-  public function testUpdateUpload()
-  {
+ * @test
+ * -# Test for UploadController::updateUpload()
+ * -# Check if response status is 202
+ */
+public function testUpdateUpload()
+{
```

Comment:
Please fix the indentation. There are lot of indentation changes here which is not required in this function. Please take a look!